test/mui_suggestors/components/mui_inline_alert_migrator_test.dart

```diff
@@ -0,0 +1,290 @@
+                  (Alert()..skin = AlertSkin.SUCCESS)();
+                  (Alert()..skin = AlertSkin.WARNING)();
+                  (Alert()..skin = AlertSkin.DANGER)();
+                }
+            '''),
+            expectedOutput: withOverReactAndWsdImports(/*language=dart*/ '''
+                content() {
+                  (mui.Alert()..severity = mui.AlertSeverity.info)();
+                  (mui.Alert()..severity = mui.AlertSeverity.success)();
+                  (mui.Alert()..severity = mui.AlertSeverity.warning)();
+                  (mui.Alert()..severity = mui.AlertSeverity.error)();
+                }
+            '''),
+          );
+        });
+
+        test('maps a gray skin to the correct variant', () async {
+          await testSuggestor(
+            input: withOverReactAndWsdImports(/*language=dart*/ '''
+                content() {
+                  (Alert()..skin = AlertSkin.GRAY)();
+                }
+            '''),
+            expectedOutput: withOverReactAndWsdImports(/*language=dart*/ '''
+                content() {
+                  (mui.Alert()..variant = mui.AlertVariant.wsdGray)();
+                }
+            '''),
+          );
+        });
+
+        test('flags the inverse skin', () async {
+          await testSuggestor(
+            input: withOverReactAndWsdImports(/*language=dart*/ '''
+                content() {
+                  (Alert()..skin = AlertSkin.INVERSE)();
+                }
+            '''),
+            expectedOutput: withOverReactAndWsdImports(/*language=dart*/ '''
+                content() {
+                  (mui.Alert()
+                    // FIXME(mui_migration) - skin prop - this prop was converted from the INVERSE skin and should be double checked
+                    ..variant = mui.AlertVariant.wsdGray)();
+                }
+            '''),
+          );
+        });
+
+        test('any other value', () async {
+          await testSuggestor(
+            input: withOverReactAndWsdImports(/*language=dart*/ '''
+                content(dynamic aValue) {
+                  (Alert()..skin = aValue)();
+                }
+            '''),
+            expectedOutput: withOverReactAndWsdImports(/*language=dart*/ '''
+                content(dynamic aValue) {
+                  (mui.Alert()
+                    // FIXME(mui_migration) - skin prop - manually migrate
+                    ..skin = aValue)();
+                }
+            '''),
+          );
+        });
+      });
+
+      group('heading', () {
+        test('is set to a string', () async {
+          await testSuggestor(
+            input: withOverReactAndWsdImports(/*language=dart*/ '''
+                content() {
+                  (Alert()..heading = 'An alert title')();
+                }
+            '''),
+            expectedOutput: withOverReactAndWsdImports(/*language=dart*/ '''
+                content() {
+                  (mui.Alert())(mui.AlertTitle()('An alert title'),);
```

Comment:
#fyi (what you have here is fine) Since the actual output and expected output are formatted before comparing them, you can format these `expectedOutput`s however you like to improve readability
```suggestion
                  (mui.Alert())(
                    mui.AlertTitle()('An alert title'),
                  );
```

Reply:
Nice, that makes sense!